addons/web/views/webclient_templates.xml

```diff
@@ -41,7 +41,7 @@
         </xpath>
         <xpath expr="//head/script[@id='web.layout.odooscript']" position="after">
             <script type="text/javascript">
-                odoo.__session_info__ = <t t-out="json.dumps(request.env['ir.http'].get_frontend_session_info())"/>;
+                odoo.__session_info__ = <t t-out="json.dumps(request.env['ir.http'].get_frontend_session_info(123))"/>;
```

Comment:
To check that test catches errors /web/login controller. Must be reverted before merging